pyccel/codegen/printing/ccode.py

```diff
@@ -1036,27 +1055,40 @@ def _print_Import(self, expr):
         if source.startswith('stc/') or source in import_header_guard_prefix:
             code = ''
```

Comment:
Does `Common_extensions` share any code with STC? If not then why not just do:
```suggestion
        if source == 'Common_extensions':
            ...
        elif source.startswith('stc/') or source in import_header_guard_prefix:
            code = ''
```
?

Reply:
You are right! It uses `common.h` but it does not need to specify anything specific to types.